system-addon/content-src/components/Sections/Sections.jsx

```diff
@@ -193,7 +193,8 @@ export class _Sections extends React.PureComponent {
   renderSections() {
     const sections = [];
     const enabledSections = this.props.Sections.filter(section => section.enabled);
-    const {sectionOrder, showTopSites} = this.props.Prefs.values;
+    const {sectionOrder} = this.props.Prefs.values;
+    const showTopSites = this.props.Prefs.values["feeds.topsites"];
```

Comment:
nit: you should be able to do `const {sectionOrder, "feeds.topsites": showTopSites} = this.props.Prefs.values;`